compiler/testSuite/14-array/fixedlength7-fe.bal

```diff
@@ -0,0 +1,3 @@
+public function main() {
+    int[true] v = [1, 2]; // @error
```

Comment:
@jclark An error is thrown at line 1 instead of 2, with `error: initializer of 'main' is not a subtype of the declared type`

Reply:
Create a bug report then.

Reply:
If the problem is related to fixed length tuples, then fix it here.